packages/esm-patient-forms-app/src/forms/forms-list.component.tsx

```diff
@@ -1,81 +1,69 @@
-import React, { useCallback, useMemo, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import {
-  DataTable,
-  DataTableSkeleton,
-  Layer,
-  Link,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableHeader,
-  TableRow,
-  TableToolbar,
-  TableToolbarContent,
-  TableToolbarSearch,
-  Tile,
-} from '@carbon/react';
-import { formatDatetime, useLayoutType, Visit } from '@openmrs/esm-framework';
-import { EmptyDataIllustration, closeWorkspace } from '@openmrs/esm-patient-common-lib';
-import { HtmlFormEntryForm } from '../config-schema';
-import { launchFormEntryOrHtmlForms } from '../form-entry-interop';
-import { useForms } from '../hooks/use-forms';
-import styles from './forms-list.scss';
 import debounce from 'lodash-es/debounce';
+import fuzzy from 'fuzzy';
+import { DataTableSkeleton, Layer, Tile } from '@carbon/react';
+import { formatDatetime, useLayoutType } from '@openmrs/esm-framework';
+import { EmptyDataIllustration } from '@openmrs/esm-patient-common-lib';
+import FormsTable from './forms-table.component';
+import styles from './forms-list.scss';
+import type { CompletedFormInfo } from '../types';
 
 export type FormsListProps = {
-  currentVisit: Visit;
-  htmlFormEntryForms: Array<HtmlFormEntryForm>;
-  patient: fhir.Patient;
-  patientUuid: string;
+  completedForms?: Array<CompletedFormInfo>;
+  error?: any;
```

Comment:
I hate using `any`, but this is meant to be directly from SWR and that's the type it gives.

Reply:
`Error` usually works for me.